docs/multichain.md

```diff
@@ -2,20 +2,190 @@
+multichain applications. One of the most important applications is a
+**treasury** which holds tokens. Managing shared assets is by far the baseline
+for establishing any working organization.
+
+Thus, for every supported chain, both the light client and treasury contracts
+are essential; Simperby provides them in a single contract while exposing an
+interface to request verification (by the light client) of the incoming message,
+for other contracts.
+
+Here are other possible use cases of Simperby-controlled contracts:
+
+1. A token that the organization issues and manages.
+2. A complex version of the treasury that might interact with other DeFi
+   services.
+3. A delegator to participate in other on-chain DAOs, on behalf of the Simperby
+   DAO.
+4. A DeFi pool whose parameters are controlled by the Simperby DAO.
+5. A bridge between two chains, where the Simperby DAO acts as the bridge
+   provider.
+
+## Universal Interoperability
+
+DAO is a technology to build an organization over decentralized networks, but
+its subject doesn't have to be limited to the blockchain. The Simperby protocol
+could govern the real-world part too, because we already have a solid way to
+deliver what's finalized by the consensus. This simple and trustless light
+client can be easily embedded as a super lightweight module in any application
+that is required to interact with a Simperby DAO. It could be a plugin attached
+to a centralized service, a webpage, or even a legal contract. There's **no need
+for a secure channel or complex authorization mechanism; it just works out of
+the box**, in a fully trustless and end-to-end verifiable way.
+
+Most of the existing blockchain protocols have light clients, but they could not
+be as simple as Simperby's because of:
+
+1. Frequent block production (Simperby produces blocks on demand)
+2. Too many consensus participants (Simperby is a permissioned blockchain)
+3. Complex consensus algorithm (Simperby uses a variation of Tendermint, which
+   is battle-tested in the Cosmos ecosystem)
+4. Non-portable implementation (Simperby uses Rust, which can be both native and
+   WASM-compiled)
+
+## DAO Spaceship
+
+Simperby is also known as **the engine for the DAO spaceship.** A DAO built with
+Simperby is completely standalone, self-hosted, and fully sovereign. It has zero
+dependency on any other blockchain or centralized service. It's like a spaceship
+floating over the space, visiting and colonizing various planets (the settlement
+chains).
+
+All you need to do is just run a super-lightweight node to maintain the
+distributed protocol. If you think it's time to contact an existing settlement
+chain, just deploy your light client and explore the ecosystem. If the
+settlement chain suddenly collapses or you want to move to another one, just
+deploy another light client and you're good to go. You can even have multiple
+settlement chains that you effectively control. No matter what happens in the
```

Comment:
```suggestion
settlement chains. No matter what happens in the
```

Reply:
I think it's worth reemphasizing the meaning of settlement chain here.